internal/virtualcircuit/retrieve.go

```diff
@@ -0,0 +1,41 @@
+package virtualcircuit
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func (c *Client) Retrieve() *cobra.Command {
+	var vcID string
+	retrieveVirtualCircuitCmd := &cobra.Command{
+		Use:     "get -i <VcId>",
+		Aliases: []string{"list"},
+		Short:   "Retrieves virtual circuit for a specific circuit Id.",
+		Long:    "Retrieves virtual circuit for a specific circuit Id.",
+		Example: `  # Retrieve virtual circuit for a specific circuit::
+
+  # Retrieve the details of a specific virtual-circuit:
+  metal interconnections get -i e9a969b3-8911-4667-9d99-57cd3dd4ef6f`,
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+
+			inc := []string{}
+			header := []string{"ID", "Name", "Port", "Speed", "Created"}
```

Comment:
`Port` should be `Type` to match the fields outputted.

Reply:
The fields that you are including here are probably sufficient for now. While it may be helpful to know the VLAN of the VirtualCircuit or the VRF CIDR, I think we can leave that to `-o json | -o yaml` for now. 

The alternative would be:

| ID | Name | Type | Speed | VLAN | VRF | VCreated
| ---|---|---|---|---|---|---
| 123 | vlan-vc | vlan | 123 | 123 | | today
| 456 | vrf-vc | vrf | 123 | | 4.5.6.7/24 | today